prime-router/src/main/kotlin/azure/HistoryFunctions.kt

```diff
@@ -477,12 +477,14 @@ open class BaseHistoryFunction {
     }
 
     fun getActionsForReportId(reportId: String, authClaim: AuthClaims): ArrayList<Action> {
-        var header: Header?
+        var header: Header? // variable is assigned but never used
```

Comment:
Instead of adding a comment just remove the variable.

Reply:
@carlosfelix2 - I told @jorg3lopez not to remove a variable if it's not being used because there are a few instances where we lock a variable on DB update.

Reply:
Ok, so remove it or leave it as is?

Reply:
@jorg3lopez leave the comment for now